front/components/assistant/TagsManager.tsx

```diff
@@ -157,7 +157,7 @@ export function TagsManager({ open, setOpen, owner }: TagsManagerProps) {
                 <DataTable data={rows} columns={columns} />
               </>
             )}
-            {rows.length === 0 && !isTagsLoading && (
+            {rows.length <= 1 && !isTagsLoading && (
```

Comment:
Why =1 ? 🙏🏻

Reply:
adding comment - now we add a default "Company" tag in migration, so by default we hve one tag and still want to see the emptyCTA